collection/rancher/v2.x/logs-collector/rancher2_logs_collector.sh

```diff
@@ -276,6 +266,7 @@ networking() {
   if $(command -v ip >/dev/null 2>&1); then
     ip addr show > $TMPDIR/networking/ipaddrshow 2>&1
     ip route show table all > $TMPDIR/networking/iproute 2>&1
+    ip neighbour > $TMPDIR/networking/ipneighbour 2>&1
```

Comment:
Do we need an ipv6 version of this?

Reply:
Added the ipv6 version.

Reply:
Should we have the same thing for the rest of the relevant commands too?

Reply:
Do you mean the IPv6 versions for the rest of the relevant commands?